src/Npgsql/Schema/DbColumnSchemaGenerator.cs

```diff
@@ -119,19 +118,19 @@ internal async Task<ReadOnlyCollection<NpgsqlDbColumn>> GetColumnSchema(bool asy
                 if (f.TableOID != 0)
                     filters.Add($"(attr.attrelid={f.TableOID} AND attr.attnum={f.ColumnAttributeNumber})");
             }
-
+				
```

Comment:
Could you please fix indentations here and below?

Reply:
Could you please be more specific about "fix indentations"?  You make a reference to line 118/119 which is the same as the original code, I did not modify it in any way.  As far as the added lines that begin at line #38, I took a swing at what I am guessing you would like to see.

Reply:
As you can see below, there are multiple tabs added. Perhaps you have whitespaces disabled on github?

![image](https://github.com/npgsql/npgsql/assets/14861420/7ba3c025-0edc-4b19-a399-387db80b983d)

Reply:
Thank you for the clarification.   I am using Visual Studio 2022 and it must have performed some automatic formatting for some reason?  I eliminated the added blanks/tabs.